Packs/Anomali_Enterprise/Integrations/Anomali_Enterprise/Anomali_Enterprise_test.py

```diff
@@ -0,0 +1,237 @@
+    }
+    mocker.patch.object(client, 'get_search_job_result_request', return_value=return_data)
+    command_results = get_search_job_result(client, args={'job_id': '111'})
+    output = command_results.to_context().get('EntryContext', {})
+    expected_result = {
+        'status': 'completed', 'category': 'forensic_api_result', 'totalFiles': 1,
+        'streamResults': [{
+            'count': '1', 'indicator': '', 'itype': '', 'severity': '',
+            'event_time': '2020-10-14T09:10:00.000+0000', 'age': '', 'event.dest': '8.8.8.8',
+            'confidence': '', 'event.src': '8.8.8.8'}],
+        'scannedEvents': 269918, 'result_file_name': 'org0_1234_job1234_result.tar.gz', 'complete': True,
+        'processedFiles': 1, 'totalMatches': 1, 'job_id': '111'
+    }
+
+    assert output.get('AnomaliEnterprise.ForensicSearch(val.job_id == obj.job_id)', []) == expected_result
+
+
+def test_get_search_job_result_command_without_matches(mocker):
+    """
+    Given:
+        - a job_id
+
+    When:
+        - mocking the server response for getting the results of a job without matches, running get_search_job_result
+
+    Then:
+        - validating the returned context data
+        - validating the returned human readable
+
+    """
+    client = Client(server_url='test', username='test', password='1234', verify=True, proxy=False)
+    return_data = {
+        'totalFiles': 0, 'streamResults': [], 'scannedEvents': 269918,
+        'complete': True, 'processedFiles': 0, 'totalMatches': 0
+    }
+    mocker.patch.object(client, 'get_search_job_result_request', return_value=return_data)
+    command_results = get_search_job_result(client, args={'job_id': '222'})
+
+    output = command_results.to_context().get('EntryContext', {})
+    expected_result = {
+        'status': 'completed', 'totalFiles': 0, 'streamResults': [],
+        'scannedEvents': 269918, 'complete': True, 'processedFiles': 0, 'totalMatches': 0, 'job_id': '222'
+    }
+    assert output.get('AnomaliEnterprise.ForensicSearch(val.job_id == obj.job_id)', []) == expected_result
+
+    hr_ = command_results.to_context().get('HumanReadable', '')
+    assert hr_ == 'No matches found for the given job ID: 222.'
+
+
+def test_get_search_job_result_command_expired_job_id(mocker):
+    """
+    Given:
+        - a job_id
+
+    When:
+        - mocking the server response for an expired job id, running get_search_job_result
+
+    Then:
+        - validating the raised error
+
+    """
+    client = Client(server_url='test', username='test', password='1234', verify=True, proxy=False)
+    return_data = {
+        'error': 'Error: Cannot find the jobId: job222'
+    }
+    mocker.patch.object(client, 'get_search_job_result_request', return_value=return_data)
+    try:
+        _ = get_search_job_result(client, args={'job_id': 'job222'})
+    except Exception as err:
+        assert 'Error: Cannot find the jobId: job222. Job ID might have expired.' in str(err)
```

Comment:
Please use [pytest.raises](https://docs.pytest.org/en/stable/reference.html#pytest-raises) or add a flag to assert the exception was raised.